includes/class-wpcom-legacy-redirector.php

```diff
@@ -13,27 +13,45 @@ class WPCOM_Legacy_Redirector {
 	const POST_TYPE   = 'vip-legacy-redirect';
 	const CACHE_GROUP = 'vip-legacy-redirect-2';
 
+	// Plugin version.
+	const PLUGIN_VERSION = '1.3.0';
+
+	// Used to flip the version of the available roles capabilities.
+	const CAPABILITIES_VER = '1.0';
```

Comment:
You're doing a numerical comparison, but have this set as a string? Use an int instead.

```suggestion
	const CAPABILITIES_VER = 1;
```

Reply:
done